src/textmate/src/bicep.ts

```diff
@@ -61,6 +61,7 @@ const keywords = [
   'type',
   'with',
   'using',
+  'extends',
```

Comment:
Could you also make this change to the other corresponding highlighting files? `src/monarch/src/bicep.ts` and `src/highlightjs/src/bicep.ts`

Reply:
done 👍